_posts/release/2020-02-29-accumulo-1.10.0.md

```diff
@@ -0,0 +1,191 @@
+action to be specified as compaction, flush or none.
+
+#### Update log statements:
+Compacting a table without user iterators was logging like it was an abnormal condition (#1347)
+Reduce verbose logging of merge operations in Master log (#1338)
+Improve logging for session expired events. When a tserver looses lock, the Session expired message
+is a log statement for every watcher - reduce current message to trace and add summary (#1108)
+
+#### Improve importdirectoy command
++ Add optional -t tablename to importdirectory shell command. (#1299)
+
+#### Add GC hadoop2 metrics reporting to expose GC  cycle metrics.
++ AccGcStarted - timestamp of GC cycle start 
++ AccGcFinished - timestamps of GC cycle finished 
++ AccGcCandidates - Number of candidates for GC 
++ AccGcInUse - Number of candidates still in use 
++ AccGcDeleted - Number of candidates deleted 
++ AccGcErrors - Number of  deletion errors 
++ AccGcWalStarted - timestamp of WAL collection start 
++ AccGcWalFinished - timestamp of WAL collection completion 
++ AccGcWalCandidates - number of WAL candidates for collection  
++ AccGcWalInUse  - number of WALs in use 
++ AccGcWalDeleted - number of WALs deleted 
++ AccGcWalErrors- number of errors during WAL deletion  
++ AccGcPostOpDuration - duration of compact / flush 
++ AccGcRunCycleCount - 1-up cycle count 
+
+#### Add option allow clone table to keep the clone offline.
++ Option to leave cloned tables offline (#1475) 
++ Updated the shell clone command with a -o option
+  
+#### Improved metadata table consistency checking and testing during GC
+    
++ During GC scans, an error will be thrown if the GC fails consistency checks (#1379)
++ Added a check to ensure the last tablet was seen.  This should cause an error to be thrown if 
+the scanner stops returning data for any reason.  
+
+### Java Lanaguage Level Changes
+This lists some of the code changes made to update the Java language level to Java 8.  They are
+listed here to aid migration of client code that may have similar issues migrating to the updated
+language level (#1467).
++ Update to maven compiler source and target to Java 1.8
++ Add optional dependency to pom for modernizer annotations
++ Add SuppressModernizer annotation to uses of Guava Predicate in AccumuloConfiguration and its 
+subclasses, Optional in VolumeChooser, and Function in GroupBalancer
++ Replace Guava Function, Predicate, Preconditions, Optional and Iterators where able with java.util equivalents
++ Replace core.util.Base64 class with java.util equivalent
++ Fix warnings introduced by Java 1.8
++ Drop method in MockConfiguration causing compiler warning and made PropertyFilter in 
+AccumuloConfiguration extend guava Predicate
++ Rename execute methods in MasterClient and ServerClient, like the 2.0 changes to fix the 
+overload warnings
++ Add colVis file to ContinuousInputFormat. Fixes #1368 (#1369)
+
+# Other Changes #
```

Comment:
The other changes could just be a list like we did here:
https://accumulo.apache.org/release/accumulo-1.9.3/#useful-links